features/convergedcloud/pkg.include

```diff
@@ -1,14 +1,15 @@
 apparmor
-open-vm-tools
+arptables
 containerd
+curl
 docker.io
+ebtables
 ethtool
-socat
-curl
-logrotate
+gnupg
 jq
-nfs-common
+logrotate
 netcat-openbsd
-ebtables
-arptables
+nfs-common
+open-vm-tools
```

Comment:
`open-vm-tools` are installed in feature `vmware`. It should only be installed by a "platform" as the image is intended to be used in a VMware environment afterwards. You may want to move it to `vmwareCC` instead.

Reply:
This list was only sorted for better readability, so I'd do this in #17